arch/arm/src/cxd56xx/cxd56_geofence.c

```diff
@@ -450,7 +450,7 @@ static void cxd56_geofence_sighandler(uint32_t data, FAR void *userdata)
       if (fds)
         {
           fds->revents |= POLLIN;
-          gnssinfo("Report events: %02x\n", fds->revents);
+          gnssinfo("Report events: %02" PRIx32 "\n", fds->revents);
```

Comment:
```suggestion
          gnssinfo("Report events: %08" PRIx32 "\n", fds->revents);
```
here and in other places